docs/examples/characterizer_example.md

```
@@ -0,0 +1,139 @@
+    sim_params.num_traj = 300  # Reducing the number of trajectories for the optimization
+
+    guess_noise_model = CompactNoiseModel(
+        [
+            {
+                "name": noise_operator,
+                "sites": [i for i in range(L)],
+                "strength": gamma_guess,
+            }
+        ]
+    )
+
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)